src/System.Private.CoreLib/shared/System/Diagnostics/SymbolStore/ISymbolDocumentWriter.cs

```diff
@@ -2,11 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
 namespace System.Diagnostics.SymbolStore
 {
     public interface ISymbolDocumentWriter
     {
-        void SetCheckSum(Guid algorithmId, byte[] checkSum);
-        void SetSource(byte[] source);
+        void SetCheckSum(Guid algorithmId, byte?[]? checkSum);
+        void SetSource(byte?[]? source);
```

Comment:
Please revert the `byte?` back to `byte` for all such cases.  There's actually a huge difference between `byte?` and `byte`... the former is actually `Nullable<byte>`.